src/sfizz/dsp/filters/filters_modulable.dsp

```diff
@@ -66,7 +66,8 @@ ap1Smooth(s,f) = fi.iir((a,1.),(a)) with {
 //------------------------------------------------------------------------------
 // Example
 //------------------------------------------------------------------------------
-// process = rbjLpfSmooth(si.smoo, cutoff, 0.0, resonance : ba.db2linear) with {
-//   cutoff = hslider("[1] Cutoff [unit:Hz] [scale:log]", 440.0, 50.0, 10000.0, 1.0);
-//   resonance = hslider("[2] Resonance [unit:dB]", 0.0, 0.0, 40.0, 0.1);
-// };
+process = rbjPeakingEqSmooth(si.smooth(ba.tau2pole(0.001)), cutoff, gain, resonance) with {
```

Comment:
Ah ok, so it was about this code. You can consider this part obsolete entirely, and remove, it was for early tests.

Beside it has an error: RBJ formulas accepts a gain specification in dB, and Q specification in linear.
From SFZ, it inputs both of them in dB. It means it was incorrect to remove `db2linear` on the parameter `resonance`.

You can generate a filter by a simpler way.
Pass the function name as `-pn`. (make it use another entry function instead of `process`)
```
faust2jack -double -pn sfzPeq src/sfizz/dsp/filters/sfz_filters.dsp
```